lang/en.json

```
@@ -66,5 +66,7 @@
   "version-downgrade-solution": "Install a newer version of Minecraft.",
   "generate-new-world-solution": "Generate a new world.",
   "bedrock-dbstorage-chain-problem": "Unable to decrypt world. Marketplace worlds can't be installed on dedicated servers.",
-  "mod-incompatible-problem": "The mods '{{first-mod-name}}' and '{{second-mod-name}}' are incompatible."
+  "mod-incompatible-problem": "The mods '{{first-mod-name}}' and '{{second-mod-name}}' are incompatible.",
+  "server-install-different-version-solution": "Install the version '{{software-version}}' of your server software.",
+  "plugin-api-version-problem": "The plugin '{{plugin-name}}' needs version '{{software-version}}' of the server software."
```

```suggestion
  "plugin-api-version-problem": "The plugin '{{plugin-name}}' needs version '{{software-version}}' of your server software."
```

I think it should be "your" in both cases then.